style.css

```
@@ -0,0 +1,204 @@
+@import "reset.css";
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700,700i,900,900i');
+* {
+  box-sizing: border-box;
+
+-webkit-font-smoothing: antialiased;
+
+-moz-osx-font-smoothing: grayscale;
+
+  }
+
+body {
+  background-color: #E5E5E5;
+}
+body > header {
+position: fixed;
+
+top: 0;
+
+  display: flex;
+
+align-items: center;
+
+justify-content: space-around;
+
+width: 100%;
+
+height: 58px;
+
+text-transform: uppercase;
+
+color: #E5E5E5;
+
+background-color: #9F9C98;
+
+font-size: 14px;
+  }
+body > header ul {
+  display: inline-flex;
+
+justify-content: space-between;
+align-items: center;
+  }
+
+.circle {
+  width: 30px;
+
+height: 30px;
+
+border-radius: 20px;
+
+background-color: #E5E5E5;
+  }
+a {
+color: inherit;
+
+  font-weight: bold;
+  }
+h1,h2,p,a,li {
+font-family: 'Roboto', sans-serif;
+}
+nav {
+  display: inline-flex;
+}
+nav div.circle {
+  width: 30px;
+
+height: 30px;
+
+color: #E5E5E5;
+
+border-radius: 20px;
+  }
+
+ h1,h2 {
+   text-align: center;
+ }
+
+h1 {
+  font-size: 28px;
+
+font-weight: bold;
+  }
+
+h2 {
+  font-size: 22px;
+
+font-weight: 100;
+  }
+
+.panel  {
+display:flex;
+
+align-items: center;
+
+justify-content: space-between;
+
+  width: 253px;
+
+height: 33px;
+  }
+.panel a {
+  text-decoration: none;
+}
+.button {
+padding: 7px 43px;
+
+  color: #fff;
+
+border-radius: 3px;
+
+background-color: #417BEA;
+  }
+.save-book {
+  color: #417BEA;
+}
+div.description  {
```

```suggestion
.description  {
```

указывать, что это div не нужно